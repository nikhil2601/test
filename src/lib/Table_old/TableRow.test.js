import React from 'react';
import { shallow } from 'enzyme';

import { TableRow } from './index';

describe('<TableRow />', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<TableRow />);
    });

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
