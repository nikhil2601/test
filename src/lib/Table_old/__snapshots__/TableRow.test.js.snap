// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableRow /> should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
